app/src/main/java/com/infomaniak/mail/data/api/ApiRoutes.kt

```diff
@@ -246,6 +246,16 @@ object ApiRoutes {
     }
     //endregion
 
+    //region SwissTransfer
+    fun swissTransferContainer(containerUuid: String): String {
+        return "$MAIL_API/api/swisstransfer/containers/$containerUuid"
+    }
+
+    fun downloadUrlSwissTransferFiles(containerUuid: String): String {
```

Comment:
The name is kinda confusing.

Maybe rename it to something like `swissTransferContainerDownloadUrl()` ?